behat-tests/features/bootstrap/FeatureContext.php

```diff
@@ -369,30 +402,21 @@ public function assertRegionElementHasClass($selector, $region, $class) {
   }
 
   /**
-   * @When I wait for messagebox to close
```

Comment:
It was not used. Want to keep it anyway? The piece is very small so I don't think its a huge inconvenience to carry it on anyway.